test/unit/serialization/compact/CompactTest.js

```diff
@@ -262,4 +263,14 @@ describe('CompactTest', function () {
         error.should.be.instanceOf(HazelcastSerializationError);
         error.message.includes('No serializer is registered for class/constructor').should.be.true;
     });
+
+    it('should throw proper error when overridding the default serializer(String class)', async function() {
```

Comment:
```suggestion
    it('should throw proper error when overriding a default serializer(String class)', async function() {
```

or "overriding the string serializer"

Reply:
Fixed